db/migrate/20231108104553_add_photo_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddPhotoToUsers < ActiveRecord::Migration[7.1]
+  def change
+    add_column :users, :photo, :string
+  end
+end
```

Comment:
**[OPTIONAL]**

- Since you're just creating the data models, you can manually edit your migration file, instead of having an extra migration file to add a single column. You can delete this file, update your migration for the user by adding this column, and then run your migration again. see what I mean here below:

` t.string :photo`

---

### PR DESCRIPTION TITLE AND BRANCH_NAME

- When adding a PR title, let's try to make it more descriptive as your current title is not very descriptive giving other users an overview of what you've done in the PR. This could be more descriptive using something like 
`Creating Data Model` and for the branch name, this could also be more descriptive by using something like 
`created_data_model || added_data_model || setup_data_model`. Please, let's make improvements on the subsequent project milestone. :muscle: